### PR TITLE
#274: [Meter] add unit test (closes #274)

### LINKED ISSUE #274: [Meter] add unit test
## Requirements

Implements tests for `Meter` component

### PR DESCRIPTION
**Pull request by nerder**
Issue #274

## Test Plan

- run `npm test` and you will see them pass

### CODE REVIEW

File: test/tests/components/Meter.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      expect(ranges).toEqual(exampleProps.ranges);
+      expect(baseLabelColor).toBe(baseLabelColor);
+      expect(baseFillingColor).toBe(baseFillingColor);
+    });
+    it('should compute className', ()  => {
+      const { className } = componentMeter.getLocals();
+      expect(className).toInclude('meter');
+      expect(className).toInclude('fancy-class-name')
+    });
+    it('should compute labelFormatter', ()  => {
```
Comment (giogonzo) [inline test/tests/components/Meter.js]: `s/labelFormatter/formattedLabel`

File: test/tests/components/Meter.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    });
+    it('should compute basisSize for decimal min-max', ()  => {
+      const meter = newComponent(Meter, {
+        value: 2.5,
+        min: 1.5,
+        max: 3.5
+      });
+      const { basisSize } = meter.getLocals();
+      expect(basisSize).toBe('50%');
+    });
```
Comment (giogonzo) [inline test/tests/components/Meter.js]: last two should be removed, they don't add anything IMO

File: test/tests/components/Meter.js (hunk 1 above)
Comment (nerder) [inline test/tests/components/Meter.js]: actually the props will test here is labelFormatter, formattedLabel is just the output

**Comment by giogonzo** [comment]
:+1:

**Pull request merged**